bin/donejs

```diff
@@ -10,7 +10,21 @@ var isWindows = require('os').platform() === 'win32';
 var utils = require('../lib/utils');
 var log = utils.log;
 
-var mypkg = require(path.join(__dirname, '..', 'package.json'));
+var mypkg = {};
+try {
+  var pkgFile = path.join(__dirname, '..', 'package.json');
+  // try to access the package.json file
+  fs.accessSync(pkgFile, fs.R_OK);
+  try {
+    mypkg = JSON.parse(pkgFile, 'utf8');
```

Comment:
This doesn't look right to me. pkgFile is going to be the path, like `path/to/package.json`, not the contents of the file.